src/components/ForecastDay.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import moment from 'moment';
+import { renderWeatherImg, toCelcius } from '../utils/weather.js';
+
+const ForecastDay = props => {
+  const { temp, weather, dt } = props.weather;
+  return (
+    <div className="forecastday">
+      <h2>{moment.unix(dt).format('ddd')}</h2>
+      <h3>{props.isCelcius ? `${toCelcius(temp.day)}° C` : `${temp.day}° F`}</h3>
+      <p>{weather[0].main}</p>
+      <i className={renderWeatherImg(weather[0].icon)} />
```

Comment:
if you're only using the first one, just pass that. unless there is a plan to iterate over these.

Reply:
done